contribs/gmf/src/directives/partials/mobilemeasurelength.html

```diff
@@ -2,29 +2,29 @@
    ng-if="ctrl.drawing && (!ctrl.valid)"
    ng-click="ctrl.addPoint()">
      <span class="fa fa-check"></span>
-     {{ ::ctrl.translate('Set as starting point') }}
+     {{::'Set as starting point' | translate}}
```

Comment:
> Any reason about the ::?

Good point. We can't use one-time-binding because otherwise the text would not be updated when changing the language. I will remove it.